{{cookiecutter.project_slug}}/requirements/local.txt

```diff
@@ -8,6 +8,9 @@ psycopg2==2.7.4 --no-binary psycopg2  # https://github.com/psycopg/psycopg2
 {%- else %}
 psycopg2-binary==2.7.4  # https://github.com/psycopg/psycopg2
 {%- endif %}
+{%- if cookiecutter.use_docker == 'y' and cookiecutter.use_celery == 'y' %}
+flower==0.9.2  # https://github.com/mher/flower
```

Comment:
Have you tested this? Incidentally, I tried to upgrade our Flower instance at work today and came across an incompatibility with Tornado v5+. It was fixed [recently in a PR](https://github.com/mher/flower/pull/793), which is merged, but not yet released.

Reply:
Just tested this out locally with a fresh project, however, I've only examined basic use cases like opening up some of the panel pages etc.

Reply:
@browniebroke as I understand we haven't got many alternatives out there anyway, unfortunatelly (at least, according to [the Celery monitoring user guide](http://docs.celeryproject.org/en/latest/userguide/monitoring.html))...

Reply:
Until the release the only option for our users would be to install the latest dev version then.

Reply:
> Just tested this out locally with a fresh project, however, I've only examined basic use cases like opening up some of the panel pages etc.

I think it was failing when trying to open Flower's home page, so a pretty basic test.

> as I understand we haven't got many alternatives out there anyway, unfortunatelly

Apologies, I realise now that my comment wasn't very obvious. I use Flower and I agree that there are not many alternatives. My comment was more directed at the version of Flower. 

> Until the release the only option for our users would be to install the latest dev version then.

I don't think we depend on Tornado, so alternatively we could pin its version to the latest v4 with a `# pyup >=4,<5`? Or wait for a newer version of Flower...

Reply:
> I think it was failing when trying to open Flower's home page, so a pretty basic test.

All the pages I've seen has been working OK, including the one for Redis broker, queues etc. I also tried it out on one of my previous projects, and hasn't spotted any problems either, at least, within the trivial scope of tasks I had at hand.

Reply:
> I don't think we depend on Tornado, so alternatively we could pin its version to the latest v4 with a # pyup >=4,<5? Or wait for a newer version of Flower...

No, we don't, and I see no reason for us to include that as a dependency only to pin its version, or, am I missing something about your statement?

Reply:
Correct, it's not needed if everything works, only to prevent getting the latest _if_ it were to cause any trouble.

Not sure why I had the problem then, maybe related to something else...

Reply:
> Correct, it's not needed if everything works, only to prevent getting the latest if it were to cause any trouble.

Does `pyup` support such semantics? :) If yes, I'm eager to take a look.

Reply:
I don't think so, I meant to explicitly declare the indirect dependency, and adding a pyup filter to it.

https://github.com/founders4schools/celery-flower-heroku/blob/master/requirements.txt

Reply:
Oh, I see. Nah, I'm feeling bad about adding any dependencies we don't actually depend on...

Reply:
On the other hand, it'd be easier to leave a TODO/NOTE whatever in the comment following the declaration, encompassing an explanatory link like the one from Flower tracker you mentioned above.